gemini/getting-started/intro_gemini_pro_python.ipynb

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
<p>Links should be to the "model-garden/" folder instead of the "getting-started/" folder</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Remain using the getting-started folder as discussed</p>